flask_saml2/sp/sp.py

```diff
@@ -37,7 +37,7 @@ class ServiceProvider:
     def login_successful(
         self,
         auth_data: AuthData,
-        relay_state: str,
+        redirect_to: str,
```

Comment:
As you've used `request.form.get('RelayState')` below, the type of this should now be `Optional[str]`